### PR TITLE
Add comments to dev-setup.sh script

### PR DESCRIPTION
**Pull request by Ichimonji10**
Add prose comments to script at `playpen/dev-setup.sh`.

### CODE REVIEW

File: playpen/dev-setup.sh [hunk 1]
```diff
@@ -1,4 +1,27 @@
...
+#    directory. These repositories are cloned via SSH, and they are cloned from
+#    the GitHub user named at the beginning of this script. This means that
+#    whoever is running this script should have a GitHub account with SSH keys,
+#    and they should have already forked the repositories into their own account.
+# 7. If no ansible inventory (a list of systems) exists at
+#    `/tmp/ansible_inventory`, then create an inventory pointing to localhost.
+# 8. Run `ansible-playbook`. Use the inventory file at `/tmp/ansible_inventory`
+#    and the playbook at `playpen/ansible/dev-playbook.yml` (in this repository).
+# 9. Set up vagrant. Do this by running the `devel/pulp/playpen/vagrant-setup.sh`
+#    shell script (in this repository).
```
Comment (rbarlow) [inline playpen/dev-setup.sh]: This script doesn't really setup Vagrant, despite the (poor) name of that script. The vagrant-setup.sh script was originally written for Vagrant to use to configure the Vagrant guest machine, which is why it has that name. This script needs to use it as well, to finish the configuration that the playbook doesn't do yet. I'm working on migrating all the code in vagrant-setup.sh to the Ansible playbook which is why I have not renamed it.

I suggest rewording this line to "Run the final configuration script, vagrant-setup.sh".

**Comment by rbarlow** [comment]
Thanks for the pull request! Just fix that one point I made a note about and let me know and I'll re-review.

File: playpen/dev-setup.sh (hunk 1 above)
Comment (Ichimonji10) [inline playpen/dev-setup.sh]: :+1:

**Comment by Ichimonji10** [comment]
Updated and rebased.

**Comment by rbarlow** [comment]
Thanks!

**Comment by Ichimonji10** [comment]
:+1:
